index.html

```diff
@@ -289,7 +289,39 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-      
+       <form
+        action="https://formspree.io/f/moqypdz" 
+        method="POST">
+  
+              <input
+                id="name"
+                name="name"
+                type="text"
+                maxlength="30"
+                placeholder="Full name"
+                required
+              />
+              <input
+                id="email"
+                name="email"
+                type="email"
+                placeholder="Email address"
+                required
+              />
+              <textarea
+                class="bg_white"
+                name="message"
+                id="message"
+                rows="5"
+                maxlength="500"
+                placeholder="Enter text here"
+                required
```

Comment:
- [ ] I will suggest you follow the given design.
- "You should stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)". Excerpt from the project requirement
- It will be nice to add the `contact me` section and the accompanying text below. So we can be 100% compliant with the design.

- Project given design: 

![image](https://user-images.githubusercontent.com/51296741/142405461-b895e8e1-bbc4-4c7c-8fec-965647fa7599.png)

- Your current project design.

![image](https://user-images.githubusercontent.com/51296741/142405748-bf13964b-476d-4131-bb7b-f46e87550d22.png)